src/X.Extensions.Logging.Telegram.Base/Formatters/DefaultLogFormatter.cs

```diff
@@ -1,9 +1,13 @@
-﻿using X.Extensions.Serilog.Sinks.Telegram.Configuration;
-using X.Extensions.Serilog.Sinks.Telegram.Extensions;
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
```

Comment:
Let's move base namespaces to global usings to avoid duplicating this. You can do it with built-in IDEs tools in couple clicks

Reply:
Done